src/data/markdown/docs/30 xk6-browser/01 xk6-browser/09 Locator.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Locator"
+excerpt: "xk6-browser: Locator Class"
+---
+
+The Locator API makes it easier to work with dynamically changing elements. Some of the benefits of using it over existing ways to locate an element (e.g. `Page.$()`) include:
+
+- Helps with writing robust tests by finding an element even if the underlying frame navigates.
+- Makes it easier to work with dynamic web pages and SPAs built with Svelte, React, Vue, etc.
+- Enables the use of test abstractions like the Page Object Model (POM) pattern to simplify and organize tests.
+- `strict` mode is enabled for all `locator` methods, which means that if more than one element matches the given selector it will throw an error.
+
+Locator can be created with the [page.locator(selector[, options])](/javascript-api/xk6-browser/page/#page-locator) method.
+
+| Method                                                                                                        | Description                                                                                                                |
+|---------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------|
+| <BWIPT id="471"/> [locator.check([options])](/javascript-api/xk6-browser/locator/check)                       | Select the input checkbox.                                                                                                 |
+| <BWIPT id="471"/> [locator.click([options])](/javascript-api/xk6-browser/locator/click)                       | Mouse click on the chosen selector.                                                                                        |
+| <BWIPT id="469"/> [locator.dblclick([options])](/javascript-api/xk6-browser/locator/dblclick)                 | Mouse double click on the chosen selector.                                                                                 |
+| [locator.dispatchEvent(type, eventInit, [options])](/javascript-api/xk6-browser/locator/dispatchevent)        | Dispatches HTML DOM event types e.g. `click`.                                                                              |
+| [locator.fill(value, [options])](/javascript-api/xk6-browser/locator/fill)                                    | Fill an `input`, `textarea` or `contenteditable` element with the provided value.                                          |
+| [locator.focus([options])](/javascript-api/xk6-browser/locator/focus)                                         | Calls [focus](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/focus) on the element, if it can be focused on. |
```

Comment:
```suggestion
| [locator.focus([options])](/javascript-api/xk6-browser/locator/focus)                                         | Calls [focus](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/focus) on the element, if it can be focused. |
```

Reply:
Resolved in: 8d0fc6bda7b29bf6b2d47d18dde49c39819fb602

Couldn't apply this directly from the review since after rebasing `docs/582-xk6-browser` the file names had to be changed.